.storybook/data/tokens.json

```diff
@@ -8,7 +8,7 @@
   "eds-typography-letter-spacing-condensed": "-0.02rem",
   "eds-typography-letter-spacing-normal": "normal",
   "eds-typography-letter-spacing-extended": "0.03rem",
-  "eds-typography-base-size": "16",
+  "eds-typography-base-size": "14",
```

Comment:
this one change triggers all the other stuff. the browser assumes "1rem = 16px" and changing the base size causes all the containers and other size units to reduce by 8.75%. font's should shrink when it doesn't have a specific `Heading` or `Text` wrapper, or inline `font-*` declarations.